test/tests/neutronics/heat_source/particles_from_postprocessor.i

```diff
@@ -0,0 +1,73 @@
+[Mesh]
+  [sphere]
+    type = FileMeshGenerator
+    file = ../meshes/sphere.e
+  []
+  [solid]
```

Comment:
we can simplify this mesh stuff - let's delete all the mesh generators from `solid` onwards (will help with maintaining this long term, and its irrelevant for the test)